docs/06_ml_regression.md

```diff
@@ -862,7 +867,13 @@ Lemmatization is, like choices around n-grams and stop words, part of data prepr
 
 ```r
 spacyr::spacy_initialize(entity = FALSE)
+```
+
+```
+#> NULL
+```
```

Comment:
I remember we had eventually gotten rid of this but now IT IS BACK 🥴

Not the end of the world, but I wonder what changed?

Reply:
I don't see. what changed. What we did in other places was to have `echo=FALSE` in the chunk. But this chunk has something we want to be printed. I'm gonna pull out the spacy call and `result='hide'` it